src/components/organisms/PageHeader/Styled.tsx

```diff
@@ -0,0 +1,289 @@
+import {ReactElement} from 'react';
+
+import {Button, DropDownProps, Dropdown, Input, Menu} from 'antd';
+
+import {
+  BellOutlined,
+  CloseCircleOutlined,
+  ClusterOutlined,
+  CopyOutlined,
+  DeleteOutlined,
+  EditOutlined,
+  FolderAddOutlined,
+  FolderOpenOutlined,
+  FormatPainterOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+
+import styled from 'styled-components';
+
+import Col from '@components/atoms/Col';
+import Header from '@components/atoms/Header';
+import Row from '@components/atoms/Row';
+
+import {AppBorders} from '@styles/Borders';
+import Colors, {BackgroundColors, FontColors} from '@styles/Colors';
+
+const {Search} = Input;
+
+export const StyledButton = styled(Button)`
```

Comment:
just to be consistent with other styled components from the code base, could you remove the "Styled" prefix from the components in this file?
then you would import them like this: `import * as S from "./Styled"` and use the components like this:
`<S.Button />` , `<S.ClusterButton />` or `<S.ClusterActionButton />` etc

note: if you want to export a `Button` component from this but it has the same name with the Antd `Button` , then you can import the button from antd like this:
`import { Button as RawButton } from "antd"`